CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+3.0.0-rc.2 Release notes (2017-??-??)
```

Comment:
This should read:
```
x.x.x Release notes (yyyy-MM-dd)
```

It gets updated with the version + date during the release process.

Reply:
Oh, that's nifty!